pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -205,26 +205,31 @@ func (this *SnapshotManager) DeleteSnapshot(peID astrolabe.ProtectedEntityID) er
 	return nil
 }
 
-func (this *SnapshotManager) DeleteProtectedEntitySnapshot(peID astrolabe.ProtectedEntityID, toRemote bool) error {
-	this.Infof("SnapshotManager.deleteProtectedEntitySnapshot Called")
-	this.Infof("Input arguemnts: peID = %s, toRemote = %v", peID.String(), toRemote)
+func (this *SnapshotManager) DeleteLocalProtectedEntitySnapshot(peID astrolabe.ProtectedEntityID) error {
```

Comment:
Let's just make this DeleteLocalSnapshot.  I'm confused between DeleteSnapshot, DeleteLocalProtectedEntitySnapshot and DeleteRemoteProtectedEntitySnapshot.
Also, let's add the PEID as a field so we know which one we are deleting here